tox.ini

```diff
@@ -5,9 +5,10 @@ minversion = 1.8
 passenv =
     HOME
 commands =
-    py.test tests {posargs}
```

Comment:
Also here, don't we need to pass the folder so that py.test just doesn't start looking into each directory?

Reply:
According to the [docs](https://docs.pytest.org/en/latest/reference.html#confval-testpaths), if we set the `testpaths` options correctly, it is possible to restrict the places where pytest searches by default, and therefore we don't need to pass the folder directly to the command...

The advantage of this approach is that one can easily overwrite the files being tested without touching `tox.ini` and `setup.cfg`.

For example, I was doing `tox -- tests/test_install.py`, to have a quick feedback and avoid running everything all the time.

Reply:
Ah, nice. Didn't know that!